apps/analysis_framework/tests/test_filters.py

```diff
@@ -21,3 +26,29 @@ def test_search_filter(self):
             expected,
             obtained
         )
+
+    def test_filter_recently_used(self):
+        af1 = AnalysisFrameworkFactory.create(title='af1')
+        af2 = AnalysisFrameworkFactory.create(title='af2')
+
+        lead = LeadFactory.create(title='one')
+        lead2 = LeadFactory.create(title='one')
+
+        EntryFactory.create(
+            analysis_framework=af1,
+            modified_at=timezone.now() - timedelta(days=90),
+            lead=lead
+        )
+
+        EntryFactory.create(
+            analysis_framework=af2,
+            modified_at=timezone.now() - timedelta(days=30),
+            lead=lead2
+        )
+
+        obtained = self.filter_class(data={'recently_used': True}).qs
+        expected = [af1, af2]
+
+        obtained_list = sorted(list(obtained), key=lambda x: x.title)
+        expected_list = sorted(list(expected), key=lambda x: x.title)
+        self.assertEqual(obtained_list, expected_list)
```

Comment:
Let's add some more data here.
```suggestion
        af1, af2, af3, _ = AnalysisFrameworkFactory.create_batch(3)

        lead = LeadFactory.create_batch(2)
        now = timezone.now()
        # Within date range window
        EntryFactory.create(
            analysis_framework=af1,
            lead=lead,
            modified_at=now - timedelta(days=90),
        )
        EntryFactory.create(
            analysis_framework=af2,
            lead=lead,
            modified_at=now - timedelta(days=30),
        )
        # Outside date range window
        EntryFactory.create(
            analysis_framework=af3,
            lead=lead,
            modified_at=now - timedelta(days=190),
        )
        # Make sure we only get af1, af2
        obtained = set(list(
            self.filter_class(data={'recently_used': True}).qs.values_list('id', flat=True)
        ))
        expected = set([af1.pk, af2.pk])
        self.assertEqual(obtained, expected)
```

Reply:
@thenav56 ,
Did you run these changes on your end ?
I get `ValueError: not enough values to unpack (expected 4, got 3)`

Reply:
@RobertKoikai No, it is just a change request with a pseudo suggestion.
Use this instead for the error you mentioned
```python3
        af1, af2, af3, _ = AnalysisFrameworkFactory.create_batch(4)
        lead = LeadFactory.create()
```